### ISSUE TITLE
PANTHER:PTN000311103 (PTHR12959) and PANTHER:PTN000319414 (PTHR13121).

### ISSUE THREAD

**Issue by sjm41** [geneontology/go-annotation#2960]
The GPI-transamidase complex (GPIT) is composed of 5 subunits (see def of GO:0042765), of which PIGK (Gpi8) is the catalytic component (e.g. see PMID:19883648 & PMID:24727937 for reviews; the introduction to PMID:28374821 is also good).

But PANTHER:PTN000311103 adds "contributes_to GPI-anchor transamidase activity" to the PIGT subunit orthologs (PTHR12959) and PANTHER:PTN000319414 adds the same to to the PIGU subunit orthologs (PTHR13121).  These are based on single annotations to the mouse/human proteins, which I'll dispute in protein2GO.  But you might want to remove them from the PAINT nodes anyway.

@hattrill

**Comment by sjm41** [comment]
_These are based on single annotations to the mouse/human proteins, which I'll dispute in protein2GO._

All source annotations have been removed.
@pgaudet - shall we close this, or do you want to leave it open so you can review the PAINT node rules directly?

**Comment by sjm41** [comment]
@pgaudet The PAINT pages have been updated and now show the correct annotations (after the source annotations were removed).  Shall we close this?

**Comment by pgaudet** [comment]
Yes it's gone. I do prefer to remove the annotations when I can, because if another annotation mistakenly comes in (say from another species), it will be automatically restored. 

Thanks, Pascale

**Comment by sjm41** [comment]
@marcfeuermann Just tagging you on directly on this one, since unlike the other I made yesterday, this one was re-opened and you were not previously assigned.
